src/components/20-molecules/datepicker/index.js

```diff
@@ -39,9 +39,7 @@ let openDatepickerInstance;
 // helper functions
 const shouldMove = elem => {
   const element = elem.getBoundingClientRect();
-  const moreSpaceOnTopThanBottom =
-    element.top > window.innerHeight - element.bottom;
-  return moreSpaceOnTopThanBottom;
+  return element.top > window.innerHeight - element.bottom;
```

Comment:
would have been nice to retain long-winded variable name as comment // more space on top than bottom!

Reply:
here the eslint rule `Local variable 'moreSpaceOnTopThanBottom' is redundant.` Then prettier format it in one line